sidecar/internal/client/client.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
```

Comment:
why not reuse the existing code from csi-lib-utils package?

Reply:
The final probe() rpc will be part of csi spec if we use that library, we need it to call csi-addons identity probe since this server will be running at csi-addons-endpoint.